.github/workflows/stable.yml

```diff
@@ -14,6 +14,11 @@ jobs:
     - name: Checkout
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
+    - name: Setup proper PHP version
+      uses: shivammathur/setup-php@9e72090525849c5e82e596468b86eb55e9cc5401 # v2.32.0
+      with:
+        php-version: 8.3
```

Comment:
So I know if I run `composer install` locally and I'm running PHP 8+, composer will pull in dependencies that rely on PHP 8+, even though our minimum is PHP 7.4 (this has caught me a few times). I don't know if this will cause the same problems, I would assume not since we currently aren't setting a specific PHP version here so I'm assuming it defaults to 8+, but worth considering so we don't end up pushing out a final release that requires PHP 8+ unexpectedly